### ISSUE TITLE
Add `joshua run` which does `joshua start` + `joshua tail`

### ISSUE THREAD

**Issue by sfc-gh-almiller** [FoundationDB/fdb-joshua#2]
It'd be nice to have a single command which can start testing and wait on the results, instead of having to (manually or via scripting) feed the ensemble ID between the two commands.

**Comment by jzhou77** [comment]
Yes, this is nice to have. A work around is the following shell script:

```
j () 
{ 
    python3 -m joshua.joshua -C path/to/cluster_file "$@"
}

name=`j start --tarball input_dir/correctness.tar.gz | tail -n 1 | awk '{print $1}'` && j tail --errors --xml | tee output_dir/$name.xml
```
